multiview/embed/splitae.py

```diff
@@ -0,0 +1,257 @@
+import torch
+from torch.utils.data import Dataset, DataLoader
+from torchvision import datasets
+import torchvision
+import matplotlib.pyplot as plt
+import PIL
+import numpy as np
+import itertools
+import tqdm
+
+from .base import BaseEmbed
+from ..utils.utils import check_Xs
+
+
+class _FullyConnectedNet(torch.nn.Module):
+    """
+    General torch module for a fully connected neural network with
```

Comment:
Since there are a lot of class parameters here can you just list them out numpy style below? a little hard to read otherwise

Reply:
True